tests/libs/build_api_v2_test.py

```diff
@@ -111,13 +116,13 @@ def test_build_summary_for_fips(
             newDeaths=FieldAnnotations(
                 anomalies=[
                     {
-                        "date": datetime.date(2020, 4, 14),
-                        "original_observation": 345.0,
+                        "date": datetime.date(2021, 3, 13),
+                        "original_observation": 110.0,
```

Comment:
Not 100% sure why these outliers changed in this way (maybe NYT corrected some of their underlying data or something).